SW_Flow_lib.c

```diff
@@ -1361,7 +1360,8 @@ void soil_temperature_init(double bDensity[], double width[], double oldsTemp[],
   int debug = 0;
   #endif
 	double d1 = 0.0, d2 = 0.0, acc = 0.0;
-	double fc_vwc[nlyrs], wp_vwc[nlyrs];
+	// double fc_vwc[nlyrs], wp_vwc[nlyrs];
+  double fc_vwc[OVER_SIZE] = {0}, wp_vwc[OVER_SIZE] = {0};

```

Comment:
Why not use existing constant for this specific purpose of soil layer number, i.e., `MAX_LAYERS`, if we want here a constant array size?